site/docs/builder.Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu
+
+RUN apt-get update && apt-get install -y git curl python3 python3-pip
```

Comment:
To keep the image small delete the apt cache again.

```
    && apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
```

Reply:
Great call, this shaved 400MB off of the docker image!